prompts.js

```diff
@@ -1,8 +1,9 @@
 module.exports = [{
-  name: `css`,
-  type: `confirm`,
-  message: `Include buefy css?`,
-  default: true
+  name: `includeBuefy`,
+  type: `list`,
+  message: `Include buefy?`,
```

Comment:
Buefy is already being included in the moment that you decide to install the plugin. The question is whether we would also like to include the style of buefy.

Reply:
Chante the message `Include buefy` to `Add buefy style`

Reply:
Change the name `includeBuefy` to `addStyle `